FoodBoard/appsettings.json

```diff
@@ -5,6 +5,7 @@
     }
   },
   "AllowedHosts": "*",
+  "SecretKey": "SuperSecretKey@345",
```

Comment:
Wie sollten einen File mit den Name KeyStore , oder ähnlich, erstellen. Die haben wir dann Lokal auf unseren Dev-Geräten. Keys sollten nicht in einem öffentlichen GitHub-Rep bleiben. Muss ich bei Android auch noch machen.

Reply:
ja gute Idee